operator/k8s_identity.go

```diff
@@ -169,5 +169,9 @@ func startManagingK8sIdentities() {
 		identityStore,
 	)
 
+	if err := waitForCRD(apiextensionsK8sClient, "ciliumidentities.cilium.io"); err != nil {
```

Comment:
Isn't `ciliumidentities` + `.cilium.io` defined anywhere?

Reply:
`ciliumidentities` is not defined anywhere as a global variable, it's defined as a local variable here:

https://github.com/cilium/cilium/blob/3a72ffc4f1808c29077d5ebb645c1b28af257996/pkg/k8s/apis/cilium.io/v2/client/register.go#L463-L479

But defining CRD names as global (const) variables is a good idea, I can do that.

Reply:
Seems like `pkg/k8s/apis/cilium.io/const.go` is a good place for global variables. I will move them there.